### PR TITLE
Add binary to scaffold new runner 

### PR DESCRIPTION
**Pull request by M4rk9696**
Adding binary as requested in #10.

Let me know if any changes are needed
@rogeliog @SimenB 

**Testing plan**
- `npm link`
- `npm init jest-runner my-runner` to use the binary

### CODE REVIEW

File: generator/fixtures/_README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# generated-jest-runner
+
+## Usage
+
+### Install
+
+Install `jest`_(it needs Jest 21+)_ and `jest-runner-eslint`
```
Comment (SimenB) [inline generator/fixtures/_README.md]: Replace `eslint`

File: generator/fixtures/_README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+module.exports = {
+  runner: 'generated-jest-runner',
+};
+```
+
+### Run Jest
+
+```bash
+yarn jest
+```
```
Comment (SimenB) [inline generator/fixtures/_README.md]: Can you add a trailing newline to the files missing it?

Comment (M4rk9696) [reply generator/fixtures/_README.md]: ok will add where ever needed

File: generator/fixtures/_README.md (hunk 1 above)
Comment (M4rk9696) [reply generator/fixtures/_README.md]: Will fix the type

File: generator/fixtures/_README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "runner": "generated-jest-runner"
+  }
+}
+```
+
+Or in `jest.config.js`
+
+```js
+module.exports = {
+  runner: 'generated-jest-runner',
```
Comment (SimenB) [inline generator/fixtures/_README.md]:
```suggestion
  runner: require.resolve('generated-jest-runner'),
```

that way it works better with yarn pnp

**Comment by SimenB** [comment]
Thank you, published!

**Pull request merged**